diem-unified-charter.md

```diff
@@ -1,12 +1,25 @@
 # Introduction
 
-Digital Emblems provide a mechanism for conveying a set of attributes
-bound to an asset that is presented to a validator and
-asserted by an issuer.
+An emblem is a device, symbol, or figure adopted and used as an identifying mark.
+As such, emblems provide a mechanism for conveying a set of attributes
+bound to an asset. An emblem may be presented to a validator and attributes are asserted by an issuer.
```

Comment:
This paragraph takes Orie's first sentence but then merges it with the existing sentence but without taking about "digital" yet. That's introduced later more explicitly to address one of the main point from the IESG review.